Examples/Python/ellipsoid_cut.py

```diff
@@ -18,15 +18,15 @@
 
 def Run_Pipeline(args):
     """
-    The data (ellipsoid.zip) is dowloaded to /Data folder and extracted to 
+    The data (ellipsoid-aligned.zip) is dowloaded to /Data folder and extracted to 
     /Output/ellpsoid_cut
     """
 
     print("\nStep 1. Extract Data\n")
     if int(args.interactive) != 0:
         input("Press Enter to continue")
     # Get data
-    datasetName = "ellipsoid-v0"
+    datasetName = "ellipsoid_aligned-v0"
```

Comment:
do we use underscore vs hyphen in datasets name? let's be consistent.

Reply:
We already have left_atrium-v0 and ellipsoid_fd-v0 so I was trying to be consistent with those. The use case .py files use underscores (left_atrium.py, ellipsoid_fd.py, deep_ssm.py, etc.) so I think the datasets were named to be consistent with that. But we don't have a use case called "ellipsoid_aligned" use case so I can change it to a hyphen if you think that's better in this case.

Reply:
This is reasonable. let's stick with underscores then. We can use hyphen only for versioning, this will be consistent with RunUseCase input arguments too.